tests/regression/run_harnesses.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
We should be able to remove this file, yeah?

Reply:
yep, it's not necessary anymore. forgot to add that to the commit